lua/entities/gmod_wire_expression2/core/selfaware.lua

```diff
@@ -203,6 +203,34 @@ e2function number entity:canSetName(string name)
 	return IsValid(this) and isOwner(self, this) and canSetName(self, this, name) and 1 or 0
 end
 
+--[[******************************************************************************]]--
+-- Extensions
+
+[nodiscard]
+e2function array getExtensions()
+	return table.Copy(E2Lib.GetExtensions())
+end
+
+local getExtensionStatus = E2Lib.GetExtensionStatus
+[nodiscard]
+e2function table getExtensionStatus()
+	local ret = E2Lib.newE2Table()
+	local s, stypes, size = ret.s, ret.stypes, ret.size
+	for _, ext in ipairs(E2Lib.GetExtensions()) do
+		s[ext] = getExtensionStatus(ext) and 1 or 0
+		stypes[ext] = "n"
+		size = size + 1
+	end
+	ret.s, ret.stypes, ret.size = s, stypes, size -- Do this just because I'm paranoid
```

Comment:
You can remove this line, only thing you need to manually set is `ret.size` because numbers aren't passed by reference in a table

Reply:
I was paranoid but thanks